python/requirements.txt

```diff
@@ -78,6 +78,7 @@ pygments
 pyspark==3.1.2
 pytest==5.4.3
 pytest-asyncio==0.16.0
+pytest-forked
```

Comment:
should not be here

Reply:
It's used in test_gcs_failure.py to make the test isolated with each other.

Reply:
do we use this elsewhere in CI? I think you can specify this in bazel instead. prefer to avoid adding a new dep